tidb-cloud/tidb-cloud-console-auditing.md

```diff
@@ -1,122 +1,170 @@
+| ends_at | timestamp | Event time |
+| operator_type | enum | Operator type: `user` or `api_key` |
+| operator_id | uint64 | Operator ID |
+| operator_name | string | Operator name |
+| operator_ip | string | Operator's IP address |
+| operator_login_method | enum | Operator's login method: `google`, `github`, `microsoft`, `email`, or `api_key` |
+| org_id | uint64 | Organization ID to which the event belongs |
+| org_name | string | Organization name to which the event belongs |
+| project_id | uint64 | Project ID to which the event belongs |
+| project_name | string | Project name to which the event belongs |
+| cluster_id | uint64 | Cluster ID to which the event belongs |
+| cluster_name | string | Cluster name to which the event belongs |
+| trace_id | string | Trace ID of the request initiated by the operator |
+| result | enum | Event result: `success` or `failure` |
+| details | json | Detailed description of the event |
 
 > **Note:**
 >
-> For scenarios where the basic fields cannot clearly describe the event, the following table shows the extended field information of these event types. Events not in the table have no extension fields.
-
-| Audit event type | Extended fields | Data type for extended fields | Description for extended fields |
-|---|---|---|---|
-| AuditEventUpdateMFA | enableMFA | bool |Enable or disable MFA |
-| AuditEventCreateProject | projectName | string | Project name |
-| AuditEventUpdateProject | oldProjectName <br/> newProjectName | string <br/> string | Old Project name <br/> New project name |
-| AuditEventDeleteProject | projectName | string | Project name |
-| AuditEventInviteUserIntoProject | email <br/> Role | string <br/> string | Email <br/> Role name |
-| AuditEventDeleteProjectUser | email <br/> Role | string <br/> string | Email <br/> Role name |
-| AuditEventUpdateOrg | orgName <br/> timeZone | string <br/> uint| Organization name <br/> Time zone |
-| AuditEventCreateIntegration | integrationType | string | Integration type |
-| AuditEventDeleteIntegration | integrationType | string | Integration type |
-| AuditEventAddNewPaymentMethod | cardNumber | string | Payment card number (information desensitized) |
-| AuditEventUpdatePaymentMethod | cardNumber | string | Payment card number (information desensitized) <br/> (Currently not getting full field information) |
-| AuditEventDeletePaymentMethod |  |  | (Currently not getting full field information) |
-| AuditEventCreateCluster | clusterName | string | Cluster name |
-| AuditEventDeleteCluster | clusterName | string | Cluster name |
-| AuditEventCreateBackup | backupName | string | Backup name |
-| AuditEventRestoreBackup | clusterName | string | Cluster name |
-| AuditEventUpdateAuditLogStatus | enableAuditLog | bool | Enable or disable database audit logging |
-| AuditEventUpdateUserRole | oldRole <br/> newRole | string <br/> string | Old role name <br/> New role name |
+> The trace_id field is temporarily unavailable
```

Comment:
How about adding this info to the last column of  `| trace_id | string | Trace ID of the request initiated by the operator |` line directly?